### PR TITLE
Deprecated plugin terser-rollup

### LINKED ISSUE #3200: Remove dependency of deprecated "rollup-plugin-terser": "^7.0.0"
Please remove dependency of "rollup-plugin-terser": "^7.0.0" plugin, as it causes warning inside the dependent applications

### PR DESCRIPTION
**Pull request by yagiz-aydin**
Deprecated plugin terser-rollup

(https://github.com/GoogleChrome/workbox/blob/v6/CONTRIBUTING.md).**

Fixes #3200 

When creating a react app with create-react-app this peer dependency gives an error.

### CODE REVIEW

**Comment by yagiz-aydin** [comment]
@tropicadri Needs PR pls

**Review by Wombosvideo — APPROVED** [review]
Changes made here are reasonable and the package is now in sync with other rollup plugins.

File: packages/workbox-build/package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "@rollup/plugin-babel": "^5.2.0",
     "@rollup/plugin-node-resolve": "^11.2.1",
     "@rollup/plugin-replace": "^2.4.1",
+    "@rollup/plugin-terser": "^0.4.3",
```
Comment (Havunen) [inline packages/workbox-build/package.json:30]: you should also delete the "rollup-plugin-terser" plugin as its replaced with "@rollup/plugin-terser"

**Comment by huineng** [comment]
is this going to be merged ?

**Comment by tropicadri** [comment]
Changes fixed in this PR: https://github.com/GoogleChrome/workbox/pull/3170

**Pull request closed without merging**